k8s-traffic-logger.py

```diff
@@ -204,15 +255,38 @@ def get_ipv6_key(k):
     ipv6_recv_bytes.clear()
 
     if ipv6_throughput:
+        pass
         # more than 80 chars, sadly.
-        print("\n%-32s %6s %6s" % ("LADDR6", "RX_B", "TX_B"))
+        # print("\n%-32s %6s %6s" % ("LADDR6", "RX_B", "TX_B"))
 
     # output
     for k, (send_bytes, recv_bytes) in sorted(ipv6_throughput.items(),
                                               key=lambda kv: sum(kv[1]),
+        
                                               reverse=True):
+        '''
         print("%-32s %6d %6d" % (
             k,
             int(recv_bytes),
             int(send_bytes),
         ))
+        '''
+
+        print("{measurement},{tag1}={val1},{tag2}={val2},{tag3}={val3},{tag4}={val4},{tag5}={val5} {field1}={field_val1},{field2}={field_val2} {timestamp}"
+                .format(measurement=MEASUREMENT,
+                        tag1="HOSTNAME",
+                        val1=hostname,
+                        tag2="IPv",
+                        val2="6",
+                        tag3="LADDR",
+                        val3=k,
+                        tag4="NAMESPACE",
+                        val4=all_pod_metadata.get(k,["",""])[0],
+                        tag5="POD",
+                        val5=all_pod_metadata.get(k,["",""])[1],
+                        field1=TX,
+                        field_val1=int(send_bytes),
+                        field2=RX,
+                        field_val2=int(recv_bytes),
+                        timestamp=int(float(time())*10**9)))
```

Comment:
Might be good to push the results to a configurable HTTP endpoint rather than printing on stdout, possibly with a configurable Authorization header.

Reply:
added. didnt have enough time to instantiate an db instance to test against but theoretically should work